ws-marketing/blog-guidelines.md

```diff
@@ -0,0 +1,38 @@
+# Inclusive Naming Initiative Blog Guidelines #
+
+## Overview ##
+
+The Inclusive Naming Initiative blog serves as a channel INI member projects to share content with the community at large, as well as a home for press releases.
+
+The content on our blog consists of:
+* Articles about projects related to the Inlcusive Naming Initiative's work 
+* Use cases and success stories from 
+
+Other topics are welcome, but it needs to have a direct link to the Inclusive Naming Initiative's work.
+
+We accept content from INI members and their organizations. The content needs to be originally published within the last 2 months.
+
+
+## Content ##
+
+The Inclusive Naming Initiative blog audience is developers, open source maintainers, and other people managing codebases. Keep this in mind as you develop your blog content.
```

Comment:
I would argue that it is much larger than this

Reply:
@xmulligan Agree... but not sure how to word it :)